include/evmc/evmc.h

```diff
@@ -626,6 +672,12 @@ struct evmc_host_interface
     /** Check account existence callback function. */
     evmc_account_exists_fn account_exists;
 
+    /** Access account callback function (EIP-2929). */
+    evmc_access_account_fn access_account;
+
+    /** Access storage callback function (EIP-2929). */
```

Comment:
Do we really need to state EIP-2929 in every place? I am fine with it for now, but perhaps in the long term we don't need it?